src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -236,6 +247,45 @@ export class AdminHomePageComponent {
     );
   }
 
+  private formatTimestampAsString(timestamp: number, timezone: string): string {
+      const dateFormatWithZoneInfo: string = 'ddd, DD MMM YYYY, hh:mm A Z';
+
+      return this.timezoneService
+          .formatToString(timestamp, timezone, dateFormatWithZoneInfo);
+  }
```

Comment:
Is it possible to use the [`FormatDetailDatePipe`](https://github.com/TEAMMATES/teammates/blob/11b8b818f3a1e2b810de20907725561b009f9297/src/web/app/components/teammates-common/format-date-detail.pipe.ts) instead of this?